app/controllers/admin/invoices_controller.rb

```diff
@@ -1,7 +1,20 @@
 class Admin::InvoicesController < ApplicationController
   def index
+    @invoices = Invoice.all.order(:id)
```

Comment:
I think the `all` is implicit here and you can call `.order` directly on `Invoice`